server/cardigan-theme/tweaks.css

```diff
@@ -0,0 +1,3 @@
+::-webkit-scrollbar {
+  width: 0.7rem;
+}
```

Comment:
Yea, tricky one.
Personally I'd like to move cardigan into it's own sub project... let's bikeshed this till then.